Doc 7956 htbt code samples

ibsoln opened the pull request:
https://issues.couchbase.com/browse/DOC-7956

DOC-7956 Replicator Heart Beat code samples

@ibsoln DOC-7956-htbt -- iOS code samples … cdeb3b3
@ibsoln DOC-7956-htbt -- Java and Android code samples … 570c8f1
@ibsoln DOC-7956-htbt -- C#/.Net code samples … dd35440

ibsoln committed:
DOC-7956-htbt -- iOS code samples
https://issues.couchbase.com/browse/DOC-7956

ibsoln committed:
DOC-7956-htbt -- Java and Android code samples
https://issues.couchbase.com/browse/DOC-7956

ibsoln committed:
DOC-7956-htbt -- C#/.Net code samples
https://issues.couchbase.com/browse/DOC-7956

ibsoln commented:
Can you please review the appropriate code samples for Replicator Heartbeat ?

- @jayahariv  iOS code samples … cdeb3b3
- @bmeike  -- Java and Android code samples … 570c8f1
- @Sandychuang8  -- C#/.Net code samples … dd35440

jayahariv commented on modules/objc/examples/code_snippets/SampleCodeTest.m:
```diff
@@ -934,6 +934,25 @@ - (void) dontTestReplicationPullFilter {
+        CBLReplicatorConfiguration* config = [self configWithTarget: target
+                                                            type: kCBLReplicatorTypePush
+                                                      continuous: YES];
+
```
can we use simple initialization for the ReplicatorConfiguration like below. The above sample function(`configWithTarget`), is internal function, if users copy paste the above, they might see build errors. 

```
CBLReplicatorConfiguration* config = [[CBLReplicatorConfiguration alloc] initWithDatabase: db target: target];
```

ibsoln replied:
@jayahariv  - see https://github.com/couchbase/docs-couchbase-lite/pull/401/commits/404291a807a98e6671c65d5000d66e147d06fa18

jayahariv commented on modules/swift/examples/code_snippets/SampleCodeTest.swift:
```diff
@@ -779,6 +779,23 @@ class SampleCodeTest {
+          self.config(target: target,
+            type: .pushAndPull,
+            continuous: true)
+
```
same like above, can we use simple config initialization?

ibsoln replied:
@jayahariv see https://github.com/couchbase/docs-couchbase-lite/pull/401/commits/109d058b0d5750d2cb325ae9bbccc5edffd527fe

ibsoln committed:
Simple initialization on ReplicatorConfiguration

https://github.com/couchbase/docs-couchbase-lite/pull/401#discussion_r570953468

ibsoln committed:
Use simple init on ReplicatorConfiguration

https://github.com/couchbase/docs-couchbase-lite/pull/401#discussion_r570954472

jayahariv commented on modules/objc/examples/code_snippets/SampleCodeTest.m:
```diff
@@ -934,6 +934,25 @@ - (void) dontTestReplicationPullFilter {
+        CBLReplicatorConfiguration* config = 
+            [[CBLReplicatorConfiguration alloc] initWithDatabase: db target: target];
+        config.type = kCBLReplicatorTypePush;
+        config.continuous: YES];
```
```suggestion
        config.continuous: YES;
```

jayahariv replied:
do you think, for consistency should we need to add comment `//  other config as required . . .` like Java and .NET?

Sandychuang8 commented on modules/csharp/examples/code_snippets/Program.cs:
```diff
@@ -1182,6 +1182,26 @@ private static void PullWithFilter(Database database)
             // end::replication-pull-filter[]
         }
 
+        public void TestReplicatorHeartbeatGetSet()
```
```suggestion
        public void TestCustomHeartbeat()
```
Change test method name to match iOS and java/android.

Sandychuang8 reviewed (approved):
Just method naming change to match with other platforms. Other than that, .Net section looks good.

ibsoln committed:
Brin function name inline with other platforms

Co-authored-by: Sandychuang8 <sandy.chuang@couchbase.com>

ibsoln committed:
Apply feedback 

per comment https://github.com/couchbase/docs-couchbase-lite/pull/401#discussion_r571324463

ibsoln commented:
Closed -- approved change now merged into feature branch Feature/DOC-7949-284-Documentation -- https://github.com/couchbase/docs-couchbase-lite/pull/407
